core/new-gui/src/app/workspace/component/workspace.component.html

```diff
@@ -1,3 +1,24 @@
+<div id="boundary">
+  <texera-menu [pid]="pid"></texera-menu>
+  <mat-expansion-panel
+    [expanded]="true"
+    hideToggle
+    cdkDrag
+    cdkDragBoundary="#boundary"
+    id="left-panel"
+    nz-resizable
+    [nzMinWidth]="175"
+    [nzMaxWidth]="screenWidth*0.5"
+    [style.width.px]="width"
+    (nzResize)="onResize($event)">
+    <mat-expansion-panel-header cdkDragHandle>
+      <mat-panel-title> Operators </mat-panel-title>
+    </mat-expansion-panel-header>
+    <texera-left-panel></texera-left-panel>
+    <nz-resize-handles [nzDirections]="['right']"></nz-resize-handles>
+  </mat-expansion-panel>
```

Comment:
can we move this to a separate component? (e.g., the left-panel). The workspace is the base of each panels, which is ideal to be like this:

```
<texera-menu ....></texera-menu>
<texera-left-panel ....></texera-left-panel>
other components...
```